master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,115 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as
+Amazon Web Services (AWS).
+
+## Why Use Calico in AWS
+
+#### Container Policy
+
+While AWS Security Groups control network traffic sent to ec2 instances, it
+does not cover containers running on hosts. Calico implements fine-grain policy
+for containers which dynamically applies to containers as they dynamically
+move around the cluster.
+
+#### No Overlays
+
+Calico minimizes the need for overlays by assigning each container
+its own IP and allowing containers to talk to one another in a pure IP network.
+
+#### Break Through the '50  Route Limit'
+
+[AWS VPC Routing tables are limited to 50 programmable routes](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Appendix_Limits.html#vpc-limits-route-tables).
+While this might limit other networking plugins, Calico supports
+ip-over-ip encapsulation to circumvent manually programming container routes in
+the VPC routing table altogether.
+
+## How to Run Calico in AWS
+
+Depending on the architecture of your AWS deployment, some of the following
+modifications may need to be made to a stock calico cluster in order for
+it to function properly.
+
+#### Allow BGP Peering in Security Groups
+
+To allow Calico containers to communicate across hosts on AWS, each host
+should have IP connectivity with one another and be able to BGP peer.
+
+To allow this BGP peering, ensure your security group settings for each
+calico host allows incoming tcp connections to port 179.
+
+You can check if your hosts have successfully established BGP sessions with
+one another using `calicoctl node status`.
+
+#### Allow Traffic Within a Single VPC Subnet
```

Comment:
It says it in the title, but otherwise this section doesn't make it clear that this will only work for traffic within a single VPC subnet and why.